File: Data Structures & Algorithms/three-integer-sum/submission-0.py
class Solution:
    def threeSum(self, nums: List[int]) -> List[List[int]]:
        ls=[]
        n=len(nums)
        sum=0
        for i in range(n):
            for j in range (i+1,n):
                for k in range(j+1,n):
                    if nums[i]+nums[j]+nums[k]==0:
                        ans=sorted([nums[i],nums[j],nums[k]])
                        if ans not in ls:
                            print(ans)
                            ls.append(ans)
        return ls
                            

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {

        unordered_map<int,int>mp;
        for(auto k:nums){
            mp[k]++;
            
        }
      for(auto it=mp.begin(); it!=mp.end(); it++){
            if(it->second>1) return true;
      }
      return false;

    }
};
